packages/shopify-codemod/test/transforms/remove-empty-returns.js

```diff
@@ -0,0 +1,14 @@
+import 'test-helper';
```

Comment:
For the test runner to pick up this file, it needs a `.test.js` suffix.  i.e., these tests aren't being executed by `npm test` :)